server/routes/model_router.ts

```diff
@@ -32,13 +32,8 @@ const validateSortItem = (sort: string) => {
 };
 
 const validateUniqueSort = (sort: string[]) => {
-  const existsSortKeyMap: { [key: string]: boolean } = {};
-  for (let i = 0; i < sort.length; i++) {
-    const [key] = sort[i].split('-');
-    if (existsSortKeyMap[key]) {
-      return 'Invalidate sort';
-    }
-    existsSortKeyMap[key] = true;
+  if (new Set(sort).size < sort.length) {
```

Comment:
Just noticed that it was checking the uniqueness of `key`, but now it's checking the entire sort string, is this change by intention?

Reply:
You are right, we should check unique sort key instead of unique sort pair. For example, `name-asc` and `name-desc` is duplicated, current implementation will ignore this.